drivers/eaton-mib.c

```diff
@@ -237,10 +245,111 @@ static info_lkp_t outlet_switchability_info[] = {
 	{ 0, NULL }
 };
 
+static info_lkp_t marlin_ambient_presence_info[] = {
+	{ -1, "unknown" },
+	{ 0, "no" },  /* disconnected */
+	{ 1, "yes" }, /* connected */
+	{ 0, NULL }
+};
+
+static info_lkp_t marlin_threshold_status_info[] = {
+	{ 0, "good" },          /* No threshold trigged */
```

Comment:
indeed, triggered and not trigged. Not sure where I inherited that one, but definitely wrong!
Fixed, thanks.